mayastor/src/bdev/nexus/nexus_bdev_rebuild.rs

```diff
@@ -71,14 +78,29 @@ impl Nexus {
             name: self.name.clone(),
         })?;
 
-        dst_child.repairing = true;
-
         job.as_client().start().context(CreateRebuildError {
             child: name.to_owned(),
             name: self.name.clone(),
         })
     }
 
+    /// Returns a `child` as an rpc Child type
+    pub fn to_child_rpc(&self, child: &NexusChild) -> Child {
```

Comment:
Maybe the function name should be "to_rpc_child" which would match the comment.

Reply:
Done